rundeckapp/grails-app/controllers/rundeck/controllers/ReportsController.groovy

```diff
@@ -298,10 +298,16 @@ class ReportsController extends ControllerBase{
         results.reports=results?.reports.collect{
             def map=it.toMap()
             map.duration= (it.dateCompleted ?: new Date()).time - it.dateStarted.time
-            if(map.executionUuid){
+            def execution=null
+            if(map.executionUuid) {
+                execution = Execution.findByUuid(map.executionUuid)
```

Comment:
Are we okay with moving these data-access operations outside of the try / catch block they were in before?

Reply:
Good catch! I will update it